static/css/material.less

```diff
@@ -14,19 +14,31 @@
   }
   .card-title {
     margin-bottom: 5px;
-    margin-top: 20px;
+    margin-top: 10px;
     padding-right: 15px;
     padding-left: 15px;
+    div, h3 {
+      display: inline-block;
+    }
+    i {
+      font-size: 2.143em;
+    }
     h3,
     h4 {
       padding-left: 0;
       margin-top: 0;
       margin-bottom: 0;
       padding-right: 10px;
-      display: inline;
     }
+    h3 {
+      font-size: 1.429em
+    }
+  }
+  .tasks-title {
+    margin-top: 20px;
+    margin-bottom: 5px;
   }
-  .tasks-count {
+  .tasks-count, tasks-title {
```

Comment:
The "tasks-title" bit isn't going to work because it needs a "." in front of it - as is it's looking for a tag of "tasks-title" not a class...